specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/OrderCreate.json

```diff
@@ -0,0 +1,113 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "orderName": "TestOrderName1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/orders/TestOrderName1?api-version=2020-12-01-preview"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "829"
+    ],
+    "orderResource": {
+      "properties": {
+        "orderDetails": {
+          "OrderType": "Purchase",
+          "expectedDataSizeInTeraBytes": 0,
+          "ProductDetails": {
+            "ProductId": "Andrew Cubone"
```

Comment:
count should be specified in the request correct? if not specified is the default = 1?

Reply:
Yes